src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Commands/BulkUploadValidateRequest/ValidateTrainingTotalHours.cs

```diff
@@ -10,41 +10,31 @@ public partial class BulkUploadValidateCommandHandler : IRequestHandler<BulkUplo
     {
         private IEnumerable<Error> ValidateTrainingTotalHours(BulkUploadAddDraftApprenticeshipRequest csvRecord)
         {
-            var domainErrors = new List<Error>();
-
-            if (!string.IsNullOrEmpty(csvRecord.TrainingTotalHoursAsString) && csvRecord.RecognisePriorLearning.Value == true)
+            if (!string.IsNullOrEmpty(csvRecord.TrainingTotalHoursAsString) && csvRecord.RecognisePriorLearning.GetValueOrDefault() == false)
+            {
+                yield return new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> cannot be set when there is no prior learning");
+                yield break;
+            }
+            
+            if (!string.IsNullOrEmpty(csvRecord.TrainingTotalHoursAsString))
             {
-                if (csvRecord.TrainingTotalHoursAsString.Contains(" "))
+                if (csvRecord.TrainingTotalHours == null)
                 {
-                    domainErrors.Add(new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> for this apprenticeship standard must be a number between 278 and 9,999"));
+                    yield return new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> for this apprenticeship standard must be a number between 278 and 9,999");
                 }
-                else if (csvRecord.TrainingTotalHoursAsString.All(char.IsDigit))
+                else if (csvRecord.TrainingTotalHours.Value > 9999)
                 {
-                    if (csvRecord.TrainingTotalHours.Value > 9999)
-                    {
-                        domainErrors.Add(new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> for this apprenticeship standard must be 9,999 hours or less"));
-                    }
-                    
-                    if (csvRecord.TrainingTotalHours.Value < 278)
-                    {
-                        domainErrors.Add(new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> for this apprenticeship standard must be 278 hours or more"));
-                    }
+                    yield return new Error("TrainingTotalHours", "Total <b>off-the-job training time</b> for this apprenticeship standard must be 9,999 hours or less");
```

Comment:
I guess this is the message from the ACs. But logically it seems it should be the same error mesage

Reply:
I think the spreadsheet show it but I have given up on the ACs